common/src/main/java/com/wynntils/core/functions/arguments/FunctionArguments.java

```diff
@@ -33,13 +34,42 @@ protected Builder(List<Argument> arguments) {
         }
 
         public ErrorOr<FunctionArguments> buildWithValues(List<Object> values) {
-            if (values.size() != this.arguments.size()) {
+            if (arguments.stream()
+                            .filter(argument -> argument instanceof ListArgument<?>)
+                            .count()
```

Comment:
Not that it really matters, but I think you can use like a findFirst or anyMatch or something like that, to short-circuit as soon as you find a ListArgument, instead of first counting up all of them. Not that it is likely to ever be more than one list argument here :-) but I think it generally makes a better pattern to try to follow. You can change this or skip this suggestion, it's up to you.

Reply:
Ah, right, you want it to be stricter greater than 1. Of course. Forget what I said.